code_challenges/adhoc/two_sum.py

```diff
@@ -0,0 +1,18 @@
+def two_sum(numbers, target):
+    j = len(numbers) - 1
+    numbers.sort()
+    for i in numbers:
+        while numbers[i] + numbers[j] > target:
```

Comment:
But this could also be a full loop couln't it?

imagine numbers = [1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,....(on billion times then), 2,3] and target = 3.